napari/settings/_fields.py

```diff
@@ -14,6 +14,10 @@ class Theme(str):
 
     # https://pydantic-docs.helpmanual.io/usage/types/#custom-data-types
 
+    def __new__(cls, v):
+        # leave as builtin string instead of coercing
+        return v
```

Comment:
In cases like this, where we use a type as a quick way to re-use validators, we have to explicitly opt out of coercion (or a `Theme` object will be passed around, which breaks some low-level things such as json-encoders down the line).

Reply:
fine... but would be better to get rid of this class entirely, as discussed offline

Reply:
I would say it doesn't actually make sense to use modify schema here.  Theme is a string whose valid values will vary at runtime based on the plugins installed... so the best  _schema_ can do is type string.  And runtime validators can certainly check the string for valid values